### ISSUE TITLE
Create new release

### ISSUE THREAD

**Issue by MatthewScholefield** [tiny-dnn/tiny-dnn#901]
Perhaps a new tag, `v1.0.0a4` should be released since it's been almost a year since the last release. The reason for this would be to allow developers to build code on the [new API](https://github.com/tiny-dnn/tiny-dnn/pull/651).

**Comment by edgarriba** [comment]
@MatthewScholefield agree. But we need to merge first part of the gsoc projects @nyanp @bhack
